gnosis/test/api-test.ini

```diff
@@ -199,9 +199,50 @@ proxy_storage:
 +requires:
     andBool POS ==Int 2
 
+; return ecrecover(hash, v, r, s);
+; bytes32 hash, uint8 v, bytes32 r, bytes32 s
+[testEcrecover]
+statusCode: _ => EVMC_SUCCESS
+callData: #abiCallData("testEcrecover", #bytes32(HASH), #uint8(SIG_V), #bytes32(SIG_R), #bytes32(SIG_S))
+callValue: 0
+; account 1 has to be active, otherwise there will be branching on <k>: #accountNonexistent(1)
+activeAccounts: SetItem(MASTER_COPY_ID) SetItem(PROXY_ID) SetItem(ORIGIN_ID) SetItem(1) _:Set
+accounts:
+    <account>
+      ; precompiled account for ECCREC
+      <acctID> 1 </acctID>
+      <balance> 0 </balance>
+      <code> .WordStack </code>
+      <storage> .Map </storage>
+      <origStorage> .Map </origStorage>
+      <nonce> 0 </nonce>
+    </account>
```

Comment:
Is this account cell general enough, or just to mock it for now?

Reply:
This is a precompiled account, so it should be an inactive account I guess. But if I leave it inactive I get issues somewhere else, don't remember where exactly. Some symbolic terms cannot be evaluated. So for the purpose of simulating a precompiled account this is general enough.

Reply:
I think we can revisit this later to make sure nothing is missed here.